app/models/client.rb

```diff
@@ -759,6 +759,536 @@ def self.sort_options(show_vispdat: false, show_assessment: false)
+      enrolled_in_so: {
+        title: 'Enrolled In SO',
+        description: nil,
+        type: 'Boolean',
+      },
+      days_literally_homeless_in_last_three_years: {
+        title: Translation.translate('Days Literally Homeless in Last Three Years'),
+        description: Translation.translate('Days in ES, SH or SO with no overlapping TH or PH'),
+        type: 'Integer',
+      },
+      requires_wheelchair_accessibility: {
+        title: 'Requires wheelchair accessibility?',
+        description: nil,
+        type: 'Boolean',
+      },
+      required_number_of_bedrooms: {
+        title: 'Minimum number of bedrooms required?',
+        description: nil,
+        type: 'Integer',
+      },
+      required_minimum_occupancy: {
+        title: 'Minimum occupancy required?',
+        description: nil,
+        type: 'Integer',
+      },
+      requires_elevator_access: {
+        title: 'Requires elevator access or ground floor unit?',
+        description: nil,
+        type: 'Boolean',
+      },
+      neighborhood_interests: {
+        title: 'Neighborhood Interests',
+        description: nil,
+        type: 'Jsonb',
+      },
+      date_days_homeless_verified: {
+        title: 'Days Homeless Verified',
+        description: nil,
+        type: 'Date',
+      },
+      who_verified_days_homeless: {
+        title: 'Who Verified Days Homeless',
+        description: nil,
+        type: 'String',
+      },
+      interested_in_set_asides: {
+        title: 'Interested in Set-Asides',
+        description: nil,
+        type: 'Boolean',
+      },
+      tags: {
+        title: 'Tags',
+        description: nil,
+        type: 'Jsonb',
+      },
+      vash_eligible: {
+        title: 'VASH Eligible',
+        description: nil,
+        type: 'Boolean',
+        display_check: 'can_view_va_eligibility?',
+      },
+      pregnancy_status: {
+        title: 'Pregnant',
+        description: nil,
+        type: 'Boolean',
+      },
+      income_maximization_assistance_requested: {
+        title: Translation.translate('Interested in income maximization services'),
+        description: nil,
+        type: 'Boolean',
+      },
+      sro_ok: {
+        title: 'Would you consider living in a single room occupancy (SRO)?',
+        description: nil,
+        type: 'Boolean',
+      },
+      dv_rrh_desired: {
+        title: 'Interested in DV Rapid Re-Housing',
+        description: nil,
+        type: 'Boolean',
+      },
+      health_prioritized: {
+        title: 'Health Prioritized',
+        description: nil,
+        type: 'Boolean',
+      },
+      is_currently_youth: {
+        title: 'Manually Flagged as Youth',
+        description: nil,
+        type: 'Boolean',
+      },
+      older_than_65: {
+        title: 'Manually Flagged as Over 65 ',
+        description: nil, 
+        ype: 'Boolean',
+      },
+      holds_voucher_on: {
+        title: 'Holds Mobile Voucher On',
+        alt_title: 'Date voucher was assigned',
```

Comment:
I could be wrong, but I feel like it is probably fine to just lean on:
```suggestion
        title: 'Date voucher was assigned',
```